index.d.ts

```diff
@@ -28,6 +28,36 @@ export interface Options {
 	```
 	*/
 	readonly maxDepth?: number;
+
+	/**
+	Indicate whether to use a `toJSON` method if encountered in the object.
```

Comment:
```suggestion
	Indicate whether to use a `.toJSON()` method if encountered in the object.
```

Reply:
I think this should explain in prose when you would want to turn it off.

Reply:
Done. Also added a link to MDN in the readme https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify#tojson_behavior